core/src/test/java/com/expediagroup/streamplatform/streamregistry/core/handler/HandlerWrapperTest.java

```diff
@@ -34,11 +35,13 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class HandlerWrapperTest {
+  private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
does this need to be a field, or could we use:

.configuration(new ObjectMapper().createObjectNode())